Packs/AzureSentinel/Scripts/MicrosoftSentinelConvertCommentsToTable/MicrosoftSentinelConvertCommentsToTable.py

```diff
@@ -1,17 +1,16 @@
 import demistomock as demisto
```

Comment:
why do we need any changes in this file at this point?

Reply:
The current script usually returns an error
and returns the irrelevant information